lib/dialects/abstract/connection-manager.js

```diff
@@ -52,7 +52,7 @@ class ConnectionManager {
   onProcessExit() {
     if (this.pool) {
       this.pool.drain(() => {
-        Utils.getLogger().debug(`connection drain via pool due to process exit`);
+        Utils.debug(`connection drain via pool due to process exit`);
```

Comment:
Yeah idea of different level of logging is good, I'll make changes to code. I'll use `const` to reduce prefix use too.